src/content/packages-and-plugins/swift-package-manager/for-plugin-authors.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Swift Package Manager for plugin authors
+description: How to add Swift Package Manager compatibility to iOS and macOS plugins
+diff2html: true
+---
+
+Flutter is migrating to Swift Package Manager to manage iOS and macOS native
+dependencies.
+This is an experimental feature that may change in the future.
+It is currently only available on the [`master` channel][].
+Flutter will continue to support CocoaPods until further notice.
+
+Swift Package Manager adoption will be gradual.
```

Comment:
if we don't know how the qualify this, not sure we need to say this. I get the sense we want to migrate the ecosystem faster than slower (but, not recklessly)

Reply:
Ah did you read this as: "Feel free to take your time, no rush on this migration"?

The intent behind this sentence was slightly different: plugin authors should **not** make plugins that only support SwiftPM. This would break users that haven't updated their projects yet. Plugin should support _both_ CocoaPods and SwiftPM until further notice.

I'll rephrase to make this clearer :)

Reply:
I tweaked this intro, let me know if it's clearer now.